Makefile

```diff
@@ -24,37 +24,3 @@ server: env
 		$(INTERFACE) \
 		--lss-address-cache-file=$(LSS_ADDRESS_CACHE_FILE) \
 		$(args)
-
-# packaging environment #######################################################
-$(PYTHON_PACKAGING_VENV)/.created: REQUIREMENTS.packaging.txt
-	rm -rf $(PYTHON_PACKAGING_VENV) && \
-	$(PYTHON) -m venv $(PYTHON_PACKAGING_VENV) && \
-	. $(PYTHON_PACKAGING_VENV)/bin/activate && \
-	pip install --upgrade pip && \
-	pip install -r REQUIREMENTS.packaging.txt
-	date > $(PYTHON_PACKAGING_VENV)/.created
-
-packaging-env: $(PYTHON_PACKAGING_VENV)/.created
-
-sdist: packaging-env
-	. $(PYTHON_PACKAGING_VENV)/bin/activate && \
-	rm -rf dist *.egg-info && \
-	./setup.py sdist
-
-_release: sdist
-	. $(PYTHON_PACKAGING_VENV)/bin/activate && \
-	twine upload dist/*
-
-# testing #####################################################################
-$(PYTHON_TESTING_ENV)/.created: REQUIREMENTS.qa.txt
-	rm -rf $(PYTHON_TESTING_ENV) && \
-	$(PYTHON) -m venv $(PYTHON_TESTING_ENV) && \
-	. $(PYTHON_TESTING_ENV)/bin/activate && \
-	pip install pip --upgrade && \
-	pip install -r ./REQUIREMENTS.qa.txt && \
-	date > $(PYTHON_TESTING_ENV)/.created
-
-qa: $(PYTHON_TESTING_ENV)/.created
-	. $(PYTHON_TESTING_ENV)/bin/activate && \
-	black --check --diff . && \
-	flake8
```

Comment:
Why are we removing these targets? 

We currently use `qa` to run all quality assurance tools before pushing to Github and get the public humiliation of the workflows finding our typo.

The `qdist`  and `_release` targets are part of the release process. As long as we do not replace those with automated releases we want to keep them.

Reply:
Hi,

I have a TODO item in the pull request description regarding these targets. I do not intend to actually remove them in the merge-ready form of this pull request, but I do not know yet how these should look in the pyproject.toml-native way of doing things. Are REQUIREMENTS.txt files still the state of the art? And what is the pypropject.toml equivalent of `./setup.py sdist`?
Both questions are likely not too hard to answer, but I did not yet get around to answering them.

Reply:
The `qa` and `build`¹ targets are back now and both are now run in CI jobs to make sure that they keep working.

¹ The `sdist` target was replaced by `build`, because it no longer uses `./setup.py sdist` but `python3 -m build`.